packages/components/src/Slider/Slider.component.js

```diff
@@ -0,0 +1,183 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classnames from 'classnames';
+import RcSlider from 'rc-slider';
+import Tooltip from 'rc-tooltip';
+import rcSliderTheme from 'rc-slider/assets/index.css'; // eslint-disable-line no-unused-vars
+import range from 'lodash/range';
+import Icon from '../Icon';
+import theme from './Slider.scss';
+
+const noFormat = value => value;
+
+/**
+ * this function check if we have icons to display
+ * @param {array} icons array if icons to display
+ */
+function isIconsAvailables(icons) {
+	return icons && Array.isArray(icons) && icons.length > 1;
+}
+
+/**
+ * This function give the selected icon position if there is more than 1 icon
+ * @param {array} icons - array of icons
+ * @param {number} value - current value of the slider
+ * @param {number} min - minimum value of the slider
+ * @param {number} max - maximum value of the slider
+ */
+export function getSelectedIconPosition(icons, value, min, max) {
+	if (isIconsAvailables(icons)) {
+		const interval = (max - min) / (icons.length - 1);
+		return Math.round(value / interval);
+	}
+	return -1;
+}
+
+/**
+ * This function allow to get the icons components
+ * @param {array} icons - array of icons
+ * @param {number} value - current value of the slider
+ * @param {number} min - minimum value of the slider
+ * @param {number} max - maximum value of the slider
+ */
+function getIcons(icons, value, min, max) {
+	if (isIconsAvailables(icons)) {
+		const position = getSelectedIconPosition(icons, value, min, max);
+		return (
+			<div className={classnames(theme['tc-slider-icons'], 'tc-slider-icons')}>
+				{icons.map((icon, index) => (
+					<Icon
+						name={icon}
+						className={index === position ? classnames(theme.selected, 'selected') : null}
```

Comment:
can you use classnames func => https://www.npmjs.com/package/classnames 
example:  classNames({ [`btn-${buttonType}`]: true });